src/lib/containers/ErrorBanner.tsx

```diff
@@ -42,7 +44,23 @@ export const ClientError = (props: { error: SynapseClientError }) => {
   if (loginError) {
     return <SignInPrompt />
   } else if (accessDenied) {
-    return <>You are not authorized to access this resource.</>
+    return (
+      <>
+        <div>You are not authorized to access this resource.</div>
+        <Button
+          variant={'tertiary'}
+          style={{ fontSize: '12px' }}
+          onClick={() => setShowDetailedError(show => !show)}
+        >
+          {showDetailedError ? 'Hide' : 'Show'} details
+        </Button>
+        <div>
+          <Collapse in={showDetailedError}>
+            <code>{error.reason}</code>
```

Comment:
I'm a little shocked that Design was ok with the code style.  I'm a little concerned that our syntax highlighter is going to find this code element and do more damage to the message style!
What do you think about putting this in an unassuming grey box instead?

Reply:
@jay-hodgson I didn't know we had a syntax highlighter for all `code` blocks. Could we change this to `pre`? I figure at some point a stack trace could end up in here so I'd like to have it preserve whitespace.